pygfx/renderers/wgpu/engine/__init__.py

```diff
@@ -92,3 +92,12 @@
 
 (Figures created with https://asciiflow.com/)
 """
+
+
+def _register_blenders():
+    # triggers an import which will allow the blenders to register
+    # themselves
+    from .blender import WgpuRenderer  # noqa F401
+
+
+_register_blenders()
```

Comment:
I don't understand the purpose of this.

Reply:
As you said. Most users don’t know what the blender code and will never import it. 

I need something to import the blender code otherwise none of the blenders get registered

Reply:
Maybe just `from . import blender   # noqa - import to register blenders` in renderer.py?

Reply:
let me try that. this introduces circular dependencies which likes to break every other version of python or something like that.